packages/jupyter-chat/src/components/input/use-chat-commands.tsx

```diff
@@ -145,7 +145,22 @@ export function useChatCommands(
         const { key, ...listItemProps } = defaultProps;
         return (
           <Box key={key} component="li" {...listItemProps}>
-            {command.name}
+            <span>
+              {command.icon !== undefined ? (
+                typeof command.icon === 'string' ? (
+                  command.icon
+                ) : (
+                  <command.icon.react />
+                )
+              ) : (
+                ''
+              )}
```

Comment:
Can you explore replacing this complex JSX expression with a local variable? This will help the code become more readable:

```tsx
const commandIcon: JSX.Element = (
  <span>
    {typeof command.icon === 'object' ? <command.icon.react /> : command.icon}
  </span>
)

return (
  <Box key={key} component="li" {...listItemProps}>
    {commandIcon}
    ...
  </Box>
)
```

"Falsy" values (e.g. `null` or `undefined`) render nothing in React, so it's easier to check if it's a `LabIcon` first, and render `command.icon` otherwise.